src/main/java/com/jw/service/OrderService.java

```diff
@@ -68,8 +68,14 @@ public String updateOrderStatusAndReturn(Long orderId) {
         return UNPROCESSED;
     }
 
+    //REVIEW-VINI: [Minor] Could we organize to have all private methods after public ones?
+    // I only put as a MINOR because looks like new conventions says that we should put methods based on functionality.
+    // I kind of dont like it, if I need to navigate to your private method to understand what is going one,
+    // then means that the method's name is bad, plus it's weird to see public and private between each other
+    // I'm trying to imagine when we add abstract methods as well :D. BUT again, it's an opinion of a grumpy dev.
```

Comment:
I see advantages from both approaches. I'll try to be consistent with public-first in the whole project, updated the class, thanks for catching that